tests/API/HcaApiTestAutomationFramework/HcaApiTests/App.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="Host" value="$(hca_hostname)" />
+    <add key="BaseUrl" value="https://$(Host)/apix/client/commerce" />
+    <add key="BaseSCUrl" value="https://$(Host)/sitecore/api/layout/render" />
+    <!-- use '$(sitecore_analytics_clobal_cookie)' e.g. "b916df89e9444a968dc1b74f53813e1e|True"-->
```

Comment:
clobal => global

Please choose one type of naming: 
`Braintree_Merchant_Id`
`BraintreePublicKey`
`SC_ANALYTICS_GLOBAL_COOKIE`